src/pretix/presale/templates/pretixpresale/event/order.html

```diff
@@ -44,6 +44,9 @@ <h2>
         {% blocktrans trimmed with code=order.code %}
             Your order: {{ code }}
         {% endblocktrans %}
+        {% if order.testmode %}
+            <span class="label label-warning">{% trans "TEST MODE" %}</span>
```

Comment:
Minor cosmetic issue: Make `display: inline-block` to match height and baseline of payment-label opposite.
![screenshot from 2019-02-19 12-59-33](https://user-images.githubusercontent.com/654548/53013492-3a2ea000-3446-11e9-870c-dc42dae56ea3.png)